client/src/http_client.rs

```diff
@@ -27,6 +27,43 @@ impl ImmuxDBHttpClient {
 }
 
 impl ImmuxDBClient<HttpClientResult> for ImmuxDBHttpClient {
+    fn get_all_groupings(&self) -> HttpClientResult {
+        let url = format!("http://{}/{}", &self.host, Constants::URL_GROUPING_KEY_WORD,);
+
+        let mut response = self.client.get(&url).send()?;
+        let status_code = response.status();
+
+        match response.text() {
+            Ok(text) => Ok((status_code, text)),
+            Err(error) => Err(ImmuxDBHttpClientError::Reqwest(error.into())),
+        }
+    }
+
+    fn remove_groupings(&self, groupings: &[GroupingLabel]) -> HttpClientResult {
+        if groupings.is_empty() {
+            return Err(ImmuxDBHttpClientError::Everything);
```

Comment:
What's `Everything`? Everything is wrong?
Also, there is nothing fundamentally wrong with passing an empty list of groupings to remove. The function just does nothing.

Reply:
I took this `Everything` from our previous code base, gonna change it now